### PR TITLE
Add premerge action

### PR DESCRIPTION
**Pull request by akuhlman-cec**
Requires some credentials to be added by an admin, will work with admins to get this added.

JIRA: INTERSIGHT-5447

### CODE REVIEW

**Comment by akuhlman-cec** [comment]
Jobs won't actually run until they're merged and another PR happens, but can see the results here: https://github.com/akulakhan/kubeless/pull/1

File: script/binary [hunk 1]
```diff
@@ -17,7 +17,7 @@
 set -e
 
 
-GIT_COMMIT=$(git describe --tags --dirty)
+GIT_COMMIT=$(git rev-parse --short HEAD)
```
Comment (akuhlman-cec) [inline script/binary:21]: fyi the old git command simply doesn't seem to work on github action builders, not entirely sure why

fails saying nothing to describe

**Review by signed8bit — APPROVED** [review]
LGTM

**Review by signed8bit — APPROVED** [review]
LGTM

**Pull request merged**